selene_sdk/sequences/genome.py

```diff
@@ -247,29 +252,15 @@ class Genome(Sequence):
     from the alphabet, but we are uncertain which.
     """
 
-    def __init__(self, input_path, blacklist_regions=None, bases_order=None):
+    def __init__(self, input_path, blacklist_regions=None, bases_order=None, init_unpickable=False):
         """
         Constructs a `Genome` object.
         """
-        self.genome = pyfaidx.Fasta(input_path)
-        self.chrs = sorted(self.genome.keys())
-        self.len_chrs = self._get_len_chrs()
-        self._blacklist_tabix = None
-
-        if blacklist_regions == "hg19":
-            self._blacklist_tabix = tabix.open(
-                pkg_resources.resource_filename(
-                    "selene_sdk",
-                    "sequences/data/hg19_blacklist_ENCFF001TDO.bed.gz"))
-        elif blacklist_regions == "hg38":
-            self._blacklist_tabix = tabix.open(
-                pkg_resources.resource_filename(
-                    "selene_sdk",
-                    "sequences/data/hg38.blacklist.bed.gz"))
-        elif blacklist_regions is not None:  # user-specified file
-            self._blacklist_tabix = tabix.open(
-                blacklist_regions)
 
+        self.input_path = input_path
+        self.blacklist_regions = blacklist_regions
+        self.initialized =False
```

Comment:
Should `initialized` be a private variable? (`_initialized`), maybe the other ones too? previously I kept a lot of things public but I'm slowly trying to update to more private variables unless we want people to access via the API